src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImplAsync.cs

```diff
@@ -2706,9 +2706,10 @@ private Task ParseEndElementAsync_CheckEndTag(int nameLen, NodeData startTagNode
                 {
 #if DEBUG
 #if !SILVERLIGHT
```

Comment:
Similarly, assuming we're never compiling with SILVERLIGHT, can we clean up these ifdefs?